### DISCUSSION TITLE
Motorcycles animation stunt/trick

### DISCUSSION THREAD

**Discussion by Burtgt** [YimMenu/YimMenu#3553] [General]
Hi there! few month ago, i saw a guy doing some tricks on his motorcycle like this : 

![23038f-bikelife1](https://github.com/user-attachments/assets/e21f0698-5324-4401-abfe-fae91883ae61)

Someone there has the animation name/dict? looking pretty cool :D!!!

**Comment by xesdoog** [comment]
You can't play vanilla animations on bikes. Well, *technically you can* but they glitch your body and only a few animations that were made for bikes can be played without glitching. Example: `veh@busted_bike@sport@stay_in_car_crim` 
So if you saw something like this in GTA Online then the player was definitely using modded rpfs with [custom animations](https://www.gta5-mods.com/misc/custom-animations-pack-1).

**Reply by Burtgt** [reply]
Thank you very much for reply xesdoog,

and in fact, the player could only play 2 or 3 animations on his motorcycle (as you said about few animations that were made for bikes), so as a biker myself in real life and in games, these few animations would have been enough for me ^^

That link about custom animations, we can use it with Yimmenu?

**Reply by xesdoog** [reply]
I'm not really sure about that since I don't load custom RPFs into online but YimMenu has the ability to do that. I think the option is labeled "Spoof Data Hashes" or something like that. Look for a pull request made by maybegreat48, it was opened just before the 1.69 update and it added that ability.

Edit: here's the [PR](https://github.com/YimMenu/YimMenu/pull/3274)
